2696. Minimum String Length After Removing Substrings/solution.py

```diff
@@ -0,0 +1,50 @@
+class Solution:
+    # Given a string of uppercase English letters, repeatedly remove
+    # any instances of "AB" or "CD" from the string until no more instances
+    # exist. Return the length of the resulting string.
```

Comment:
I think it's unclear that removing "AB"s and "CD"s can generate more.

Reply:
The explanation is on line 39.

Reply:
Wouldn't it be better to explain it now?